src/components/table/table.scss

```diff
@@ -76,6 +80,14 @@ $m-table--icon-vertical-offset: -5px !default;
     td {
         padding: $m-spacing;
         box-sizing: content-box;
+
+        &.m--is-centered {
+            text-align: center;
+
+            & * {
+                text-align: center;
+            }
```

Comment:
Pourquoi le * si tu appliques déjà strictement ton style à un td ou un th ayant une class m--is-centered ?

Est-ce que ce que tu tentes de faire ne serait-il pas plutôt ceci:

`            > * {
                text-align: center;
            }`

Reply:
Je veux que ça soit aussi à tous les enfants

Reply:
dans ce cas tu dois enlever le & si tu veux targeter les enfants, ou utiliser ma suggestion si tu veux targeter les enfants directs